### ISSUE TITLE
Commands are sensitive to double-whitespace

### ISSUE THREAD

**Issue by viraptor** [radare/radare2#6043]
Some commands will not work correctly with multiple spaces instead of one, but will not report an error/warning either. For example:

```
// this works
f foobar=0x1000
// this does nothing
f  foobar=0x1000
```

version 0.10.6

**Comment by Maijin** [comment]
? well if you type garbage it's expected to output garbage don't really see what's the problem here tbh

**Comment by viraptor** [comment]
If it's garbage, I expect radare to warn: "this is garbage, ignoring". It already implements that in some cases:
```
[0x00409e20]> )
|ERROR| Invalid command ')' (0x29)
```
So why not others?

**Comment by radare** [comment]
because / is a command

**Comment by Maijin** [comment]
@radare I don't think you read the issue.

**Comment by radare** [comment]
the way its done now this should be fixed in every single command with r_str_chop_ro(). i have some ideas to generalize all that stuff, but its not gonna land in 1.0

**Comment by radare** [comment]
i have fixed the issue you comment in here in a597684970e70dfadebff379ee3c9ede5510b327 if you found other inconsistenccies please let us know. also, having a test case for this is important. can you send a pullreq to r2r adding the "f   foo" test? to confirm that adding spaces is not a problem to add new flags?

thx

**Comment by Maijin** [comment]
https://github.com/radare/radare2-regressions/commit/5a9e4a744b28aac57258cb31bb2806cec809c81a
